src/index.js

```diff
@@ -1,41 +1,113 @@
 // import _ from 'lodash';
```

Comment:
Kindly add a new module to your repository for the new functionalities as requested on project requirements.
<img width="437" alt="Screenshot 2023-02-01 at 11 23 21 PM" src="https://user-images.githubusercontent.com/25887253/216155742-21be0ef2-0d79-4bce-be9b-c4d3b5781fad.png">